### ISSUE TITLE
Option to exit pdfpc by "falling off" the last slide

### ISSUE THREAD

**Issue by multixrulz** [pdfpc/pdfpc#649]
I am the author of [revelation](https://bitbucket.org/multixrulz/revelation/src/master/), a presentation program that doesn't really overlap with pdfpc's function at all (but does need documentation and a website).

I would like to integrate the display of PDFs into this program, and pdfpc seems to be the perfect way to do it, with one problem: I want it to exit by clicking the "forward" button (i.e. page-down) on the last page of the PDF.  This is because the way I use revelation is to set up the whole slideshow  - which is a combination of various pre-built presentations - and then run it away from the computer using a presenter clicker.  The way pdfpc is now, I'd have to walk back to the computer and hit Ctrl-Q in order to exit and continue with the rest of my slideshow.

Could this be made a command-line option?

**Comment by fnevgeny** [comment]
Would a `pdfpcrc (5)` option do? We already have `black-on-end`, so the new one would probably be `exit-on-end`.

**Comment by multixrulz** [comment]
TLDR; A pdfpcrc option will do the job.

Long version:
I wasn't even aware of the pdfpcrc file until you pointed it out.

My approach with external programs is to not require special configuration by the user in order for them to work, so my immediate answer is "no, a pdfpcrc option won't do".  But in practice, it probably will, and the need/ability to set up a pdfpcrc file can be documented in revelation.

Have you considered allowing a commandline argument to specify which pdfpcrc file to use - ie another place to look in addition to the system-wide and user files?  That way a user could bundle the pdf, pdfpc, and pdfpcrc files and have a completely portable configuration.

**Comment by fnevgeny** [comment]
> Have you considered allowing a commandline argument to specify which pdfpcrc file to use - ie another place to look in addition to the system-wide and user files?

As mentioned in the `pdfpcrc` man page, the location of the user file is `$XDG_CONFIG_HOME/pdfpc/pdfpcrc`. Hence, you can (re)define the `$XDG_CONFIG_HOME` envvar when running `pdfpc`. `$XDG_CONFIG_HOME/pdfpc/pdfpc.css` is also the place where you can adjust visual style details, perhaps making its look more native to your `revelation` envelope.

**Comment by multixrulz** [comment]
That makes sense.  I keep forgetting that Unix had solutions to all sorts of problems that modern software just keeps reinventing badly.

**Comment by multixrulz** [comment]
Oops, still need that option i the pdfpcrc file.
